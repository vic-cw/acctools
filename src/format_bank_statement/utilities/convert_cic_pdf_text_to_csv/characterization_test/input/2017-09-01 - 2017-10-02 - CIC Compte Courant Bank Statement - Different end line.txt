                                                                        Crédit Industriel et Commercial

                                                                                                       CIC PARIS SAINT MARTIN

                                                                                           224 RUE SAINT MARTIN 75010 PARIS
                                                    ¤     u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                                   12345@cic.fr                                     BIC : CMCIFRPP

                                                                                                                                                                                                                    45678

                                                                                                                                                                                                                    12345

                                                                                                                                   M  LOUIS  MALLE

                                                                                                                                   1  RUE  DU   CHEMIN

                                                                                                                                   75020   PEKIN

...........................................................................................................................V..O...T.R..E...C..O..N..S..E..I.L..L..E..R..:..M...S..é..b..a.s..t.i.e.n...T..S..E..U...........................

                                              RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                Agence 12345

                                              02 octobre 2017

                                                    C/C CONTRAT PERSONNEL GLOBAL N° 00012345678 en euros (GD)

                                                    TITULAIRE(S) : M LOUIS MALLE

                                              Date        Date valeur   Opération                                                                         Débit EUROS                                         Crédit EUROS

                                                                        SOLDE CREDITEUR AU 31/08/2017                                                                                                         0,01                                                                                       Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                              01/09/2017  01/09/2017    PROUT                                                                                                                       5,52

                                              Information sur la protection des comptes :                                                                                                                     <<Suite au verso>>

                                              (GE) : protégé par la Garantie de l'Etat, (GD) : protégé par la Garantie des Dépôts

                                              Remarque : Le livret réglementé relève de la garantie de l'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des Dépôts.

                                              www.garantiedesdepots.fr

                                                                                                                                                                                                              Page 1

                                                CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                        M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                        Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                          RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                 Agence 12345

                                          02 octobre 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                                    en  euros  (GD)

                                                TITULAIRE(S) : M LOUIS MALLE

                                          Date        Date valeur   Opération                                                                   Débit EUROS                                                Crédit EUROS

                                          07/09/2017  07/09/2017    TRANSFER                                                                                                                    1.000,00

                                          Information sur la protection des comptes :

                                          (GE) : protégé par la Garantie de l'Etat, (GD) : protégé par la Garantie des Dépôts

                                          Remarque : Le livret réglementé relève de la garantie de l'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des Dépôts.

                                          www.garantiedesdepots.fr

                                                                                                                                                                                                           Page 2
                                                                        Crédit Industriel et Commercial

                                                                                                           CIC PARIS SAINT MARTIN

                                                                                           224 RUE SAINT MARTIN 75010 PARIS
                                                    ¤     u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                              12345@cic.fr                                          BIC :      CMCIFRPP

                                              RELEVE ET INFORMATIONS BANCAIRES                                                                                                                                 Agence 12345

                                              02 octobre 2017

                                                    C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                                    en  euros  (GD)

                                                    TITULAIRE(S) : M LOUIS MALLE

                                              Date        Date valeur        Opération                                                              Débit EUROS                                                Crédit EUROS

                                              15/09/2017  15/09/2017    INCOMING FUNDS                                                                                                                         1,00

                                                                        LINE TWO

                                              Information sur la protection des comptes :                                                                                                                      <<Suite au verso>>

                                              (GE) : protégé par la Garantie de l'Etat, (GD) : protégé par la Garantie des Dépôts

                                              Remarque : Le livret réglementé relève de la garantie de l'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des Dépôts.

                                              www.garantiedesdepots.fr

                                                                                                                                                                                                               Page 3

                                                CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                             M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                        Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
                                          RELEVE ET INFORMATIONS BANCAIRES                                                                                                                               Agence 12345

                                          02 octobre 2017

                                                C/C CONTRAT PERSONNEL GLOBAL N° 00012345678                                         en  euros  (GD)

                                                TITULAIRE(S) : M LOUIS MALLE

                                          Date        Date valeur   Opération                                                                        Débit                                      EUROS    Crédit EUROS

                                          21/09/2017  21/09/2017    TWO LINE EXPENSE                                                                                                            10,23

                                                                    NATURALIA          CARTE 00123456

                                          Réf : 009                 SOLDE CREDITEUR AU 02/10/2017                                                                                                        0,01

                                          QXBAN : AB01 CMCI FRPP XXX CDE2 FGHI 3JKL MNOP 4QR                                            IBAN : FR76 0123 4567 8910                                       1112 1314 151

                                            SITUATION DE VOS CREDITS en euros au 02 octobre 2017

                                          Numéro                                       Libellé du compte                                       Montant emprunté                                          Solde (*)

                                          00012345678               ALLURE LIBRE                                                        EUR                                                     +0,00  +0,00

                                          (*) Solde correspondant à l'amortissement normal du prêt

                                          Information sur la protection des comptes :

                                          (GE) : protégé par la Garantie de l'Etat, (GD) : protégé par la Garantie des Dépôts

                                          Remarque : Le livret réglementé relève de la garantie de l'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des Dépôts.

                                          www.garantiedesdepots.fr

                                                                                                                                                                                                         Page 4
                                                                        Crédit Industriel et Commercial

                                                                                                     CIC PARIS SAINT MARTIN

                                              ¤                                                     224 RUE SAINT MARTIN 75010 PARIS
                                                 u0820 81 23 45 (Service 0,12 /min + prix appel) FAX 01 44 16 81 39
                                                                                                                                                12345@cic.fr                                        BIC  :  CMCIFRPP

                                              RELEVE ET INFORMATIONS BANCAIRES                                                                                                                              Agence 12345

                                              02 octobre 2017

                                              AUTORISATIONS DE DECOUVERTS

                                              N° de compte                                 Libellé                                                                                                          Montant euros

                                              00012345678               DECOUVERT C. PERS. GLOBAL TAEG 15,12% AU                   29/08/2017                                                                         0,00

                                              TOTAL MENSUEL DES FRAIS DEBITES SUR PRODUITS                                         ET SERVICES  BANCAIRES

                                              N° de compte                                 Libellé                                                                                                          Montant euros

                                              00012345678               TOTAL DES FRAIS EN AOU 2017                                                                                                                   14,58

KV.20171002.012345.6789.1011.12131 X 0 R                                                                                                                                                                                                   d'une réclamation, appelez le 09 69 32 06 06 appel non surtaxé  Pour tout appel en vue d'obtenir la bonne exécution d'un contrat ou le traitement

                                              Information sur la protection des comptes :

                                              (GE) : protégé par la Garantie de l'Etat, (GD) : protégé par la Garantie des Dépôts

                                              Remarque : Le livret réglementé relève de la garantie de l'Etat et le(s) livret(s) ordinaire(s) de dépassement associé(s) de la garantie des Dépôts.

                                              www.garantiedesdepots.fr

                                                                                                                                                                                                            Page 5

                                              CREDIT INDUSTRIEL ET COMMERCIAL (CIC) - Banque régie par les articles L.511-1 et suivants du Code Monétaire et Financier - SA au capital de 608 439 888 

                                          6,  avenue de Provence 75009 Paris - Adresse postale : 75452 Paris cedex 09 - tél 01 45 96 96 96 - swift CMCIFRPP - www.cic.fr - 542 016 381 RCS Paris - TVA intracommunautaire : FR34542016381

                                                                        M le Médiateur du CIC : 63 chemin Antoine Pardon 69160 Tassin la demi lune - www.lemediateur-cic.fr

                                                                        Pour les opérations effectuées en sa qualité d'intermédiaire en opérations d'assurance : ORIAS 07 025 723 (www.orias.fr)
