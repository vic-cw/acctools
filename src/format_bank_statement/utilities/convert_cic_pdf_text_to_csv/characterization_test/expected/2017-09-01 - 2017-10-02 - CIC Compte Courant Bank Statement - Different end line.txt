2017-09-01,2017-09-01,"PROUT","-5,52"
2017-09-07,2017-09-07,"TRANSFER","-1.000,00"
2017-09-15,2017-09-15,"INCOMING FUNDS LINE TWO","1,00"
2017-09-21,2017-09-21,"TWO LINE EXPENSE NATURALIA          CARTE 00123456","-10,23"
